OneBlink.SDK/models/FormElement.cs

```diff
@@ -224,6 +229,10 @@ public string hint
         {
             get; set;
         }
+        public HintPositionEnum hintPosition

```

Comment:
this needs to be a string so that when we add a new type, any applications that are using this version of the SDK dont break when the API returns a string that is not in the enum.